techmatters/flex-plugins
# ft: configure lex for e2e account

Primary reviewer: @stephenhand 

## Description

- New feature: Migrate E2E account from Autopilot to Lex
- Task: Configure lex for e2e account on Terraform.

### Checklist

### Related Issues
Fixes [CHI-2213](https://tech-matters.atlassian.net/browse/CHI-2213)

### Verification steps

Run `make HL=e2e HL_ENV=development plan` and verify everything is fine

[CHI-2213]: https://tech-matters.atlassian.net/browse/CHI-2213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
